platform.md

```diff
@@ -807,6 +829,18 @@ The lifecycle MAY provide other mechanisms by which a platform can supply regist
 
 The lifecycle MUST attempt to authenticate anonymously if no matching credentials are found.
 
+### Experimental Features
+
+Where noted, certain features are considered experimental and susceptible to change in a future API version.
+
+The platform SHOULD set `CNB_EXPERIMENTAL_MODE=<warn|error|silent>` in the lifecycle's execution environment to control the behavior of experimental features.
+
+When an experimental feature is invoked, the lifecycle:
+- SHALL warn and continue if `CNB_EXPERIMENTAL_MODE` is unset
+- SHALL warn and continue if `CNB_EXPERIMENTAL_MODE=warn`
+- SHALL fail if `CNB_EXPERIMENTAL_MODE=error`
+- SHALL continue without warning if `CNB_EXPERIMENTAL_MODE=silent`
```

Comment:
I understand that this is influenced by [this RFC](https://github.com/buildpacks/rfcs/blob/main/text/0066-lifecycle-prelease-version-and-experimental-section.md#experimental-features-in-the-api) but given that `CNB_PLATFORM_EXPERIMENTAL_FEATURES` isn't included and there's no way to "enable" experimental features I wonder if this really shouldn't default to `error`. Otherwise, it seems like we might run into future situations where a new behavioral experimental feature would have no real way of being enabled (or worse, be enabled by default). If we had `CNB_EXPERIMENTAL_MODE` default to `error` and a platform passes a flag and tries to use experimental behavior, it would "fail early" forcing them to understand that they are using an experimental feature by explicitly requiring them to set the `CNB_EXPERIMENTAL_MODE` accordingly.

Thoughts?

Reply:
I wholeheartedly agree with this logic. I'll make this change... maybe we should propose an "amendment" to the linked RFC.